src/background/sidebar-injector.js

```diff
@@ -438,4 +433,16 @@ export function SidebarInjector() {
       args: [clientConfig],
     });
   }
+
+  /**
+   * @param {number} tabId
+   * @param {number} [frameId]
+   */
+  async function executeClientBootScript(tabId, frameId) {
+    return executeScript({
+      tabId,
+      frameId,
```

Comment:
I assume Chrome ignores the `frameId` if the property is set but `undefined`?

Reply:
Yes